docs/src/pages/getting-started/installation/troubleshooting.react.mdx

```diff
@@ -40,3 +43,13 @@ export default defineConfig({
     "skipLibCheck": true,
 ...
 ```
+
+**4.** Update the `package.json` file and add `@aws-amplify/ui` to `resolutions`:
+
+```js
+...
+  "resolutions": {
+    "@aws-amplify/ui": "^3.0.0"
+  },
+...
```

Comment:
Does this work the same in npm and yarn?

Reply:
I thought only `yarn` supported resolutions...

Reply:
Yeah, I only tested it in `yarn`, didn't know it was yarn specific. Do ya'll think I should include a disclaimer of using `yarn` or remove it entirely since it's not universal? At the moment it's a stopgap measure to fix Vite applications until we remove the dependency on `@aws-amplify/ui@v2` from `aws-amplify`.

Reply:
Removed for now